pom.xml

```diff
@@ -67,67 +67,8 @@
     <url>https://www.apache.org</url>
   </organization>
 
-  <developers>
-    <developer>
-      <id>vinothchandar</id>
-      <name>Vinoth Chandar</name>
-      <organization>Confluent Inc</organization>
-    </developer>
-    <developer>
-      <id>prasannarajaperumal</id>
-      <name>Prasanna Rajaperumal</name>
-      <organization>Snowflake</organization>
-    </developer>
-    <developer>
-      <id>n3nash</id>
-      <name>Nishith Agarwal</name>
-      <organization>Uber</organization>
-    </developer>
-    <developer>
-      <id>bvaradar</id>
-      <name>Balaji Varadharajan</name>
-      <organization>Uber</organization>
-    </developer>
-  </developers>
-
-  <contributors>
-    <contributor>
-      <name>Wei Yan</name>
-      <organization>Uber</organization>
-    </contributor>
-    <contributor>
-      <name>Siddhartha Gunda</name>
-      <organization>Uber</organization>
-    </contributor>
-    <contributor>
-      <name>Omkar Joshi</name>
-      <organization>Uber</organization>
-    </contributor>
-    <contributor>
-      <name>Zeeshan Qureshi</name>
-      <organization>Shopify</organization>
-    </contributor>
-    <contributor>
-      <name>Kathy Ge</name>
-      <organization>Shopify</organization>
-    </contributor>
-    <contributor>
-      <name>Kaushik Devarajaiah</name>
-      <organization>Uber</organization>
-    </contributor>
-    <contributor>
-      <name>Anbu Cheeralan</name>
-      <organization>DoubleVerify</organization>
-    </contributor>
-    <contributor>
-      <name>Jiale Tan</name>
-      <organization>Vungle</organization>
-    </contributor>
-  </contributors>
-
   <inceptionYear>2016</inceptionYear>
```

Comment:
remove this too?  can you go line-by-line and see whats left? :) thats the only way to clean this up

Reply:
@vinothchandar : Saw this in flink https://github.com/apache/flink/blob/master/pom.xml#L36. thats why left it.

Reply:
Will make a pass for others

Reply:
@vinothchandar : Took a look at the parent apache pom (https://github.com/apache/maven-apache-parent/blob/apache-21/pom.xml) and made changes accordingly to hoodie parent pom